src/Security/Authorization/Core/src/Microsoft.AspNetCore.Authorization.csproj

```diff
@@ -13,13 +13,7 @@ Microsoft.AspNetCore.Authorization.AuthorizeAttribute</Description>
   </PropertyGroup>
 
   <ItemGroup>
-    <Compile Include="$(SharedSourceRoot)SecurityHelper\**\*.cs" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <Reference Include="Microsoft.AspNetCore.Authentication.Abstractions" />
     <Reference Include="Microsoft.AspNetCore.Http.Abstractions" />
```

Comment:
One of the prerequisites for this working for client-side Blazor is removing the `Microsoft.AspNetCore.Http.Abstractions` reference. Will that be possible as part of your refactoring? It was something I did in my alternate PR #9997.

If you don't want to do that here, then I can do it in a follow-up PR.

Reply:
Yep there was no ref to http in 2.2, should be safe to remove